example/src/example_layer.cpp

```
@@ -34,31 +39,156 @@ bool TestLayer::OnEvent(moth_ui::Event const& event) {
+                             m_target->GetImage() != nullptr &&
+                             m_target->GetImage()->GetWidth() > 0 &&
+                             m_target->GetImage()->GetHeight() > 0;
+    if (targetValid) {
+        m_graphics.SetTarget(m_target.get());
+
+        m_graphics.SetBlendMode(canyon::graphics::BlendMode::Replace);
+        m_graphics.SetColor({ 0.05f, 0.05f, 0.15f, 1.0f });
+        m_graphics.Clear();
+
+        // Eight spinning coloured lines radiating from the centre
+        float const cx = 128.0f, cy = 128.0f, radius = 110.0f;
```

Clang (14.0.6)

[error] multiple declarations in a single statement reduces readability

(readability-isolate-declaration,-warnings-as-errors)